source/code/plugins/changetracking_lib.rb

```diff
@@ -50,6 +50,21 @@ def self.packageXMLtoHash(packageXML)
         ret
     end
 
+    def self.fileInventoryXMLtoHash(fileInventoryXML)
+        fileInventoryHash = instanceXMLtoHash(fileInventoryXML)
```

Comment:
that's true, however we will be using this in very near future. if we get it in now, we wont have to touch this code later, we will only need mof changes.
However, I am open to removing it, if you feel very strongly about it.